offchain-modules/packages/app-multisign-server/src/sigServer.ts

```diff
@@ -68,18 +68,19 @@ export class SigServer {
 
     if (ForceBridgeCore.config.ckb !== undefined) {
       const ckbKeys = new Map<string, string>();
-      ForceBridgeCore.config.ckb.multiSignKeys.forEach((key) => {
-        ckbKeys[key.address] = key.privKey;
-      });
+      const ckbPrivateKey = ForceBridgeCore.config.ckb.privateKey;
+      const ckbAddress = privateKeyToCkbAddress(ckbPrivateKey);
+      ckbKeys[ckbAddress] = ckbPrivateKey;
       SigServer.keys['ckb'] = ckbKeys;
     }
-    if (ForceBridgeCore.config.eth.multiSignKeys !== undefined) {
+    if (ForceBridgeCore.config.eth !== undefined) {
       const ethKeys = new Map<string, string>();
-      ForceBridgeCore.config.eth.multiSignKeys.forEach((key) => {
-        ethKeys[key.address] = key.privKey;
-      });
+      const ethPrivateKey = ForceBridgeCore.config.eth.privateKey;
+      const ethAddress = privateKeyToEthAddress(ethPrivateKey);
+      ethKeys[ethAddress] = ethPrivateKey;
       SigServer.keys['eth'] = ethKeys;
     }
+    logger.info(`sigKeys: ${JSON.stringify(SigServer.keys, null, 2)}`);
```

Comment:
we should NOT print private key in logger

Reply:
it is used to debug, forget to delete, fixed.